src/components/PublicProfile.vue

```diff
@@ -118,6 +215,22 @@
           overallRating: (overallRating / Object.keys(userMetrics).length).toFixed(1),
         };
       },
+      stepOneNewReview: function() {
```

Comment:
I would change this to use object method shorthand:
stepOneNewReview() { ...
per Airbnb's JS style guide: https://github.com/airbnb/javascript#es6-object-shorthand